tasks/parameter_context_tasks.yml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Idk if the code here can be made more DRY.  We have two blocks of code at `13-19` and `30-36` that do the same (ish) thing.

Reply:
hmm yea maybe nested loops, but I don't want to make this gross